app/pages/dashboard/controller.test.js

```diff
@@ -6,6 +6,7 @@ jest.mock('../../helpers/api/ordapi/getOrders');
 jest.mock('./contextCreator', () => ({
   getContext: jest.fn(),
 }));
+jest.mock('../select/controller');
```

Comment:
Can we update the the test `'should call getContext with the correct params when orders data is returned by getData'` to include the `userIsProxy` change?